testing/internal/e2e/boundary/scope.go

```diff
@@ -45,7 +47,7 @@ func CreateNewOrgCli(t testing.TB, ctx context.Context) string {
 	output := e2e.RunCommand(ctx, "boundary",
 		e2e.WithArgs(
 			"scopes", "create",
-			"-name", "e2e Org",
+			"-name", fmt.Sprintf("e2e Org %s", time.Now().String()),
```

Comment:
**Note:**
This is adjusted to make it easier to execute the test against local environment multiple times.
Storage Buckets cannot be deleted once there has been a session recorded and the Orgs cannot be deleted if they have a used Storage Bucket.
The test creates new organization. As we cannot delete it after the test, the organization has to have a unique name.

Reply:
What if, for now, we set the `-scope-id` to `global`?